Makefile

```diff
@@ -18,6 +18,10 @@ test:
 		go tool cover -html coverage.out -o coverage.html; \
 		exit $$ret)
 
+.PHONY: bench
+bench:
+	go test ./... -bench . -benchtime 5s -timeout 0 -run=XXX -cpu 1 -benchmem
```

Comment:
Some of the tests bench concurrent access. I don't think we need `-run=XXX -cpu 1`

Reply:
Ahh OK. I'll remove `-cpu 1` but keep `-run` since that is supposed to ensure that all benchmarks in all folders run

Reply:
nit: I've used `-run '^$'` for this purpose to match an empty string which is (probably) impossible.

Reply:
I don't know what i'm doing wrong, but i get an error

```
[21/11/23 4:36:35] ~/GitHub/gubernator (run-benchmarks) $ make bench
go test ./... -bench . -benchtime 5s -timeout 0 -run '^ -benchmem
/bin/sh: -c: line 0: unexpected EOF while looking for matching `''
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [bench] Error 2
[21/11/23 4:36:44] ~/GitHub/gubernator (run-benchmarks) $ 
```